js/dashboard-demo/tts.js

```diff
@@ -0,0 +1,156 @@
+/**
+ * Created by mgl on 03.06.19.
+ */
+const textToSpeechLimit = 30;
+var socket;
+
+/**
+ *@function connect Connects the user html page to the server, connects the client to WebSocket, subscribes client to notifications
+ */
+function connect() {
+  socket = new SockJS('http://mgladki.cern.ch:8080/fake-notifications');
+  stompClient = Stomp.over(socket);
+  stompClient.connect({}, function (frame) {
+    console.log(typeof(frame));
+    console.log('Connected: ' + frame);
+    stompClient.subscribe('/topic/notification', function (content) {
+      console.log(content.body);
+      var body = JSON.parse(content.body);
+      playOrDelay(body.filename, body.text);
+      console.log(body.filename);
+      console.log(body.text);
+    });
+  });
+}
+
+/**
+ *@function playOrDelay plays or not a notification if there is already a notification playing
```

Comment:
If you comment the function you don't use the @function and the name of the function. Please look how the functions are commented in :
https://jsdoc.app/about-getting-started.html